frontend/javascripts/oxalis/model/helpers/nml_helpers.ts

```diff
@@ -721,6 +722,7 @@ export function parseNml(nmlString: string): Promise<{
               isVisible: _parseFloat(attr, "color.a") !== 0,
               groupId: groupId >= 0 ? groupId : DEFAULT_GROUP_ID,
               type: _parseTreeType(attr, "type", TreeTypeEnum.DEFAULT),
+              edgesAreVisible: _parseBool(attr, "edgesAreVisible", true),
```

Comment:
this is only for parsing right? what about serialization? can be invoked via the trees-tab ("save selected trees as nml" or sth like that)

Reply:
You are right, I did not take care of serialization. Internally we agreed to postpone the support of the new `edgesAreVisible` field in nmls. Thus I did not implement any NML support besides what is needed for typescript. This line is needed because the created object `currentTree` in line `712`, because the field `edgesAreVisible` is required by this object. Alternatively, I could set the value to always be true, if you prefer this variation over already trying to parse a value that is not yet serialized, I would also be fine with that :)

Reply:
Ah, I see, makes sense :)